src/libYARP_dev/src/modules/ControlBoardWrapper/ControlBoardWrapper.h

```diff
@@ -327,6 +327,7 @@ class yarp::dev::ControlBoardWrapper:   public yarp::dev::DeviceDriver,
     int               top;          // to be removed
     int               period;       // thread rate for publishing data
     bool              _verb;        // make it work and propagate to subdevice if --subdevice option is used
+    std::vector<JointTypeEnum>     jointTypes;
```

Comment:
This is another change in the ABI that could be avoided if we were using pimpl here.
I suggest that later we move all the private part of this class into an opaque pointer (in `devel`).